05-26 13:55:41 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:25:10 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:25:50 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:26:09 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:27:24 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:30:13 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:32:49 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:33:41 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:34:25 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:36:00 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-26 14:39:10 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:88)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
